internal/exec/stages/files/units.go

```diff
@@ -33,6 +34,7 @@ type Preset struct {
 	enabled        bool
 	instantiatable bool
 	instances      []string
+	path           string
```

Comment:
I'd be inclined to bind the path at a lower level of the code, i.e. make this field `scope` rather than `path`.

Reply:
done